public/mockServiceWorker.js

```diff
@@ -8,111 +8,111 @@
  * - Please do NOT serve this file on production.
  */
 
-const INTEGRITY_CHECKSUM = '3d6b9f06410d179a7f7404d4bf4c3c70'
-const activeClientIds = new Set()
+const INTEGRITY_CHECKSUM = '3d6b9f06410d179a7f7404d4bf4c3c70';
+const activeClientIds = new Set();
 
```

Comment:
This file starts with:
```javascript
/* eslint-disable */
/* tslint:disable */
/**
 * Mock Service Worker (1.2.3).
 * @see https://github.com/mswjs/msw
 * - Please do NOT modify this file.
 * - Please do NOT serve this file on production.
 */
```

This file should probably be completely ignored by prettier also so it doesn't try to change anything in this file.

Reply:
fixed